Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/Map.cs

```diff
@@ -31,8 +31,8 @@ public class Map {
             }
         }
 
-        public IReadOnlyCollection<Police> Police => Players.Where(user => user is Police).Select(user => new Police(user)).ToList();
-        public IReadOnlyCollection<Thief> Thiefs => Players.Where(user => user is Thief).Select(user => new Thief(user)).ToList();
+        public IReadOnlyCollection<Police> Police => Players.Where(user => user is Police).Select(user => (Police) user).ToList();
```

Comment:
Ik weet niet of `(Police) user` gaat werken. Volgensmij doen we op andere plekken `new Police(user)`, omdat er anders iets niet goed gaat. Toch? Wat is er anders verandert waardoor het nu wel goed gaat?

Reply:
Als het goed is werkt dit gewoon. Ben er tijdens het testen niet tegenaan gelopen. 
Als dit wel met new zou moeten zou Thief ook wat gegevens missen, daarom had ik het in casts veranderd.

Reply:
Wordt de `Police` wel ergens gebruikt? Want ik had ook in de applicatie ergens `(Thief) user` maar toen kwamen alleen de properties terug die in de class `Thief` staan en niet vanuit de abstracte class van de `Player`. Kan je dit daarom is testen?

Reply:
Police wordt niet gebruikt nee. 

Dit werkt voor mij wel gewoon.
De properties van Player staan in de debugger onder "base":

![image](https://user-images.githubusercontent.com/10811551/120003641-3c451900-bfd6-11eb-8aca-163aa77dc7d3.png)